tgui/packages/tgui/interfaces/GasCanister/Detonator.js

```diff
@@ -0,0 +1,210 @@
+import { Fragment } from 'inferno';
+import { Button, LabeledList, Section, Box, Flex, Divider } from '../../components';
+import { DetonatorTimer } from './DetonatorTimer';
+
+export const Detonator = props => {
+  const {
+    detonator,
+    detonatorAttachments,
+    onToggleAnchor,
+    onToggleSafety,
+    onWireInteract,
+    onPrimeDetonator,
+    onTriggerActivate,
+    onSetTimer,
+  } = props;
+
+  return (
+    <Section
+      title="Detonator">
+      <DetonatorWires
+        detonator={detonator}
+        onWireInteract={onWireInteract}
+        onSetTimer={onSetTimer} />
+      <Divider />
+      <DetonatorUtility
+        detonator={detonator}
+        detonatorAttachments={detonatorAttachments}
+        onToggleAnchor={onToggleAnchor}
+        onToggleSafety={onToggleSafety}
+        onPrimeDetonator={onPrimeDetonator}
+        onTriggerActivate={onTriggerActivate} />
+    </Section>
+  );
+};
+
+const DetonatorWires = props => {
+  const {
+    detonator: {
+      wireNames,
+      wireStatus,
+      time,
+      isPrimed,
+    },
+    onWireInteract,
+    onSetTimer,
+  } = props;
+
+  return (
+    <Flex>
+      <Flex.Item>
+        <LabeledList>
+          { wireNames.map((entry, i) => (
+            <LabeledList.Item
+              key={entry + i}
+              label={entry}>
+              <Box
+                height={1.7}>
+                { (wireStatus && wireStatus[i]) ? (
+                  <Fragment>
+                    <Button
+                      icon="cut"
+                      content="Cut"
+                      onClick={() => onWireInteract("cut", i)} />
+                    <Button
+                      icon="bolt"
+                      content="Pulse"
+                      onClick={() => onWireInteract("pulse", i)} />
+                  </Fragment>)
+                  : (
+                    <Box
+                      color="average"
+                      minHeight={1.4}>
+                      Cut
+                    </Box>) }
+              </Box>
+            </LabeledList.Item>
+          ))}
+        </LabeledList>
+      </Flex.Item>
+      <Flex.Item
+        mr={5}
```

Comment:
This seems like a large, fairly arbitrary margin. Do we really need `5em` (what `5` gets converted to) of right margin on this?

Reply:
I wanted it to be much closer to left-most components but I think we can get away with `2`